content/v2.0/reference/cli/influx/config/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: influx config – Config management commands
+description: The 'influx config' command and subcommands manage multiple InfluxDB connection configurations.
+menu:
+  v2_0_ref:
+    name: influx config
+    parent: influx
+weight: 101
+v2.0/tags: [config]
+---
+
+The `influx config` command manages multiple InfluxDB connection configurations in the `config` file (by default, stored at `~/.influxdbv2/config`). Each connection includes a url, token, associated organization (`org`), and active setting.
```

Comment:
- ~url~ -> URL
- ~('org`)~

I'd either make them all inline code using the actual keys from the file, or all regular english.